docs/README.md

```diff
@@ -26,92 +26,183 @@
-  - [ONNX](custom_operations.md#onnx)
-- [Client Libraries and Examples](https://github.com/triton-inference-server/client)
-  - [C++ HTTP/GRPC Libraries](https://github.com/triton-inference-server/client#client-library-apis)
-  - [Python HTTP/GRPC Libraries](https://github.com/triton-inference-server/client#client-library-apis)
-  - [Java HTTP Library](https://github.com/triton-inference-server/client/tree/main/src/java)
-  - GRPC Generated Libraries
-    - [go](https://github.com/triton-inference-server/client/tree/main/src/grpc_generated/go)
-    - [Java/Scala](https://github.com/triton-inference-server/client/tree/main/src/grpc_generated/java)
-    - [Javascript](https://github.com/triton-inference-server/client/tree/main/src/grpc_generated/javascript)
-- Performance Analysis
-  - [Performance Tuning Guide](performance_tuning.md)
-  - [Optimization](optimization.md)
-  - [Model Analyzer](model_analyzer.md)
-  - [Performance Analyzer](perf_analyzer.md)
-  - [Inference Request Tracing](trace.md)
-- [Jetson and JetPack](jetson.md)
-
-## Developer Guide
-The Developer Guide describes how to build and test Triton and also how Triton can be extended with new functionality.
-
-- [Build](build.md)
-- [Protocols and APIs](inference_protocols.md).
+# **Triton Inference Server Documentation**
+
+| [Installation](README.md#installation) | [Getting Started](README.md#getting-started) | [User Guide](README.md#user-guide) | [API Guide](protocol/README.md) | [Additional Resources](README.md#resources) | [Customization Guide](README.md#customization-guide) |
+| ------------ | --------------- | --------------- | ------------ | --------------- | --------------- | 
+
+## **Installation**
+Before you can use the Triton Docker image you must install
+[Docker](https://docs.docker.com/engine/install). If you plan on using
+a GPU for inference you must also install the [NVIDIA Container
+Toolkit](https://github.com/NVIDIA/nvidia-docker). DGX users should
+follow [Preparing to use NVIDIA
+Containers](http://docs.nvidia.com/deeplearning/dgx/preparing-containers/index.html).
+
+Pull the image using the following command.
+
+```
+$ docker pull nvcr.io/nvidia/tritonserver:<yy.mm>-py3
+```
+
+Where \<yy.mm\> is the version of Triton that you want to pull. For a complete list of all the variants and versions of the Triton Inference Server Container,  visit the [NGC Page](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver). More information about customizing the Triton Container can be found in [this section](customization_guide/compose.md) of the User Guide.
+
+## **Getting Started**
+
+This guide covers the simplest possible workflow for deploying a model using a Triton Inference Server.
+- [Create a Model Repository](getting_started/quickstart.md#create-a-model-repository)
+- [Launch Triton](getting_started/quickstart.md#launch-triton)
+- [Sending an Inference Request](getting_started/quickstart.md#sending-an-inference-request)
```

Comment:
Nit for consistency: "Send an Inference Request"

Reply:
+1